### ISSUE TITLE
Unneeded

### ISSUE THREAD

**Issue by WubTheGame** [Lordmau5/Carry-Stacker-Reloaded#9]
This was originally to ask for BLT 2.1 support. However, Thanks to enragedpixel i no longer need this issue.

**Comment by enragedpixel** [comment]
@WubTheGame please check my fork, I am now maintaining this mod as of BLT 2.0, please submit any bugs/issues there, enjoy!

**Comment by WubTheGame** [comment]
Ok Thanks!

**Comment by WubTheGame** [comment]
Wait, I don't see any releases, How am i supposed to use it?


.....Wait, i just download the source code thing as a zip and put it in my mods folder don't i?

**Comment by enragedpixel** [comment]
sorry had forgotten some commits so re download it, but yes, download the zip remove all "-"'s and "master" and you're good to go @WubTheGame

**Comment by WubTheGame** [comment]
Ok i'll try if i can understand that.

**Comment by WubTheGame** [comment]
Got it going, Thanks!

**Comment by WubTheGame** [comment]
@enragedpixel I want to submit a bug report however there is no "issues" page on your fork repo so i will just post it here:

When you get downed, you can revive yourself by simply throwing a bag. This works without the skill you would normally need to have.

Ways this Bug can impact your game:
You are alive with an empty health and shield bar.
You can revive yourself with only a bag.
after getting downed again you skip bleedout mode.
Anything else i may not have found.

I have tested and recreated this multiple times after experiencing it for the first time and i am sure that it's reproducable.
From what i have tested this works on multiplayer if you are host or in singleplayer.

This bug also works with or without Swan Song, but you have to be in bleedout mode or just downed in order for the glitch to work.

TL;DR Do a heist, Get a bag, Get downed and throw your bag. Boom, glitch.

It does work when:
You are in single player or are the host of a multiplayer heist.
You are in bleedout mode
You are out of bleedout mode but still not fully alive or in custody.

It does not work when:
You have the skill Swan Song and the skill is currently in effect.
You do not have a bag to throw.
You are not the host.

**Comment by enragedpixel** [comment]
issues page has been added, please repost this there and delete it from here, i doubt lordmau5 wants all the emails from a mod he's no longer maintaining

**Comment by Lordmau5** [comment]
Not even getting any emails, had that disabled a LONG time ago :)

Also, you create a file called `ISSUE_TEMPLATE.md` in the main directory of the repository that will then automatically be put into a new issue for people to fill out:
https://github.com/blog/2111-issue-and-pull-request-templates

**Comment by WubTheGame** [comment]
Thanks Lordmau5.

**Comment by enragedpixel** [comment]
@Lordmau5 Thanks for the tips and putting up with my inexperience with github
